pumps/resurface.go

```diff
@@ -74,10 +79,26 @@ func (rp *ResurfacePump) Init(config interface{}) error {
 		rp.log.Info(rp.GetName() + " Initialized (Logger disabled)")
 		return errors.New("logger is not enabled")
 	}
+	rp.initWorker()
 	rp.log.Info(rp.GetName() + " Initialized")
 	return nil
 }
 
+func (rp *ResurfacePump) initWorker() {
+	rp.data = make(chan []interface{}, 5)
```

Comment:
@monrax could I ask you again what's the reason why the size of this channel is hardcoded to 5?

Reply:
hi @mativm02 , sorry for the late reply -- Good catch! Ideally, it would be 1 instead of 5. Although, after some load testing I found this to be a good queue depth. I do agree it would be better to make it configurable by defining a new parameter for this pump backend. Please, let me know if this is a priority for y'all and I'll make sure to submit a new PR with this change. Thanks!